fixtures/flows/brief/training.js

```diff
@@ -1,230 +1,235 @@
-export const create = async (params) => {
-    console.log('Starting to create training brief');
-    await createBrief();
-    await fillTitle(params.title);
-    await fillOrganisation();
-    await fillWhatTraining();
-    await fillLds();
-    await fillTrainingDetail();
-    await fillWhyTraining();
-    await fillAudience();
-    await fillTrainingLength();
-    await fillTrainingMethod();
-    await fillHowLong();
-    await fillWhoCanRespond();
-    await fillContactNumber();
-    await fillCommencementDate();
-    await fillLocation(params.locations);
-    await fillBudgetRange();
-    await fillPaymentApproach();
-    await fillSecurityClearance();
-    await fillContractLength();
-    await fillIntellectualProperty();
-    await fillAdditionalTerms();
-    await fillNumberOfSellers();
-    await fillEvaluationRating();
-    await fillTechnicalCompetenceCriteria();
-    await fillCulturalFitCriteria();
-    await fillAssessmentMethods(params.evaluations);
-    await fillSummary();
-    await fillQuestionAnswer();
-    await publishBrief();
+import * as util from '../../flows/utils'
+
+/* eslint-disable no-await-in-loop */
```

Comment:
is this still needed? if so can it be put closer to where the line is with `eslint-disable-next-line`